src/app/app.css

```diff
@@ -245,6 +245,42 @@ html, body, #root {
   overflow: auto;
 }
 
+.dashboard-card-resizable-wrapper { 
+  display: flex;
+  justify-content: space-between 
+}
+
+.pf-c-card.pf-m-rounded.dashboard-card {
```

Comment:
Probably remove this? Card has default border-radius 3px. This should already match with that we use for resize bar.

Reply:
The card border radius is only acting on the card part but the resizable bar is another div that extends beside it meaning that it isn't part of the card. See here if I turn it off: 
![image](https://user-images.githubusercontent.com/69827283/214994327-49604c89-b5f3-477d-8919-2ae717972115.png)